build.gradle.kts

```diff
@@ -21,6 +21,7 @@ ext["mapstruct.version"] = "1.5.5.Final"
 ext["postgresql.version"] = "42.6.0"
 ext["kotlin.logging.version"] = "3.0.5"
 ext["springdoc.openapi.version"] = "2.1.0"
+ext["wiremock.version"] = "3.0.0-beta-10"
```

Comment:
Unfortunately the non-beta stream of wiremock (version 2.xx) no longer works with Spring Boot 3.
You have to use a version `3.0.0-beta-2` or above 😢 
`3.0.0-beta-10` is the latest and [based on the frequency of beta releases](https://mvnrepository.com/artifact/com.github.tomakehurst/wiremock) this is clearly active development so I'd like to think there is a formal v3 release soon 🤞